contracts/ServiceManagerToken.sol

```diff
@@ -70,6 +114,39 @@ contract ServiceManagerToken is GenericManager, OperatorSignedHashes {
         emit OperatorWhitelistUpdated(newOperatorWhitelist);
     }
 
+    /// @dev Changes Identity Registry Bridger contract address.
+    /// @param newIdentityRegistryBridger New Identity Registry Bridger contract address.
+    function changeIdentityRegistryBridger(address newIdentityRegistryBridger) external {
```

Comment:
No we don't need that imo. We simply deploy a new periphery contract. Keep it simple stupid